src/components/common/layout/languageToggle.njk

```diff
@@ -0,0 +1,18 @@
+<nav class="language-select" aria-label="Choose a language">
+    <ul class="govuk-list">
+        {% for lang in ['en', 'cy'] %}
+            <li class="govuk-header__navigation-item">
+                {% set translationKey = 'general.languageToggle.' + lang + '.linkText' %}
+                {% if language == lang %}
+                    <span class="govuk-!-font-size-16" aria-current="true">{{ translationKey | translate }}</span>
+                {% else %}
+                    <a aria-current="page"
```

Comment:
I might be getting myself confused as this is a part of ARIA I'm not super familiar with but why should non-active items have `aria-current="page"`? 🤔 

The guidance on `aria-current` seems to indicate that a value of `page` should represent that we are currently on the current page within a set of pages.

On the other hand a value of `true` seems a bit more general: represents the current item within a set.

Based on these descriptions I would probably lean towards using "true" as opposed to "page" for the active element. But whether it's "true" or it's "page", I think that only the currently selected option should have the `aria-current` attribute, where the non-selected option(s) shouldn't have `aria-current` at all (or have the attribute, but with a value of "false")

Reply:
That's not quite what I was questioning with my comment. I could have written it more clearly.

I think using a span as the active item is fine. The other existing examples of this component use span – not to say that we should indiscriminately copy what other teams in government have done, just saying that there is precedent, and if teams like HMRC and GOVUK are using it in their live service, it's _probably_ okay. 

I also think adding `aria-current="true"` to it is a good idea as it represents the currently active item within a set of items.

What I am not so sure about is why/if the un-selected language should have `aria-current` at all given that it's not the current item.